README.md

```diff
@@ -40,8 +36,16 @@ Create a new SearXNG instance in five minutes using Docker
 > (Get-Content searxng/settings.yml) -replace 'ultrasecretkey', $secretKey | Set-Content searxng/settings.yml
 > ```
 
-> [!NOTE]  
-> On the first run, you must remove `cap_drop: - ALL` from the `docker-compose.yaml` file for the `searxng` service to successfully create `/etc/searxng/uwsgi.ini`. This is necessary because the `cap_drop: - ALL` directive removes all capabilities, including those required for the creation of the `uwsgi.ini` file.
+### Method 1: With Caddy included (recommended for beginners)
+6. Check everything is working: `docker compose up`
+7. Run SearXNG in the background: `docker compose up -d`
+
+### Method 2: Bring your own reverse proxy (experienced users)
+6. Remove the caddy related parts in `docker-compose.yaml` such as the caddy service and its volumes.
+7. Point your reverse proxy to the port set for the `searxng` service in `docker-compose.yml` (8080 by default). You can change the port `searxng` listens on inside the docker container (e.g. if you want to operate in `host` network mode) with the `BIND_ADDRESS` environment variable (defaults to `0.0.0.0:8080`). You can just set the environment variable inside `docker-compose.yaml`.
```

Comment:
There are too many things in one single point. Please try to split them. or say like:

- the solutions:
  - solution A
  - solution B